fold-and-r.csl

```diff
@@ -119,8 +119,8 @@
         </else-if>
         <else-if type="article-journal">
           <group delimiter=" ">
-            <text macro="title" prefix="'" suffix="'"/>
-            <text term="in" suffix=" "/>
+            <text macro="title" quotes="true"/>
+            <text term="in" prefix=", " suffix=" "/>
```

Comment:
What I have above (if placed in the group) would be one solution, i.e. put the comma in the suffix. When you want the comma in the delimiter, you'd need

```
<group delimiter=", ">
  <text macro="title" quotes="true"/>
  <group delimiter=" ">
    <text term="in"/>
    <text macro="journal"/>
  </group>
</group>
```

Hope that makes sense. The latter is considered better style.